internal/handlers/telegram/helpers.go

```diff
@@ -5,12 +5,25 @@ import (
 )
 
 /*
-GetUsername returns the name and username of a user. Since usernames are optional
+ResolveUserName does basic cleanup if a user does not have a username on Telegram.
+*/
+func ResolveUserName(u *tgbotapi.User) string {
```

Comment:
Since this method inserts a ZWSP in the username, I think the function name and comment should be more clear that the character is inserted.

Also an aside, but this is probably a good place to insert a boolean check later, where if TeleIRC is configured to have ZWSP off, it will not be inserted?

Reply:
@jwflory oh, you're right! I completely forgot to add the boolean with the ZWSP check. I will get to that tomorrow.

Reply:
@Tjzabel Cool. This is the last thing on my feedback list before this is good to merge. :+1: